pocs/tests/test_ioptron.py

```diff
@@ -3,24 +3,25 @@
 from astropy.coordinates import EarthLocation
 
 from pocs.mount.ioptron import Mount
-from pocs.utils.config import load_config
 
 
+@pytest.mark.with_mount
 def test_loading_without_config():
     """ Tests the basic loading of a mount """
     with pytest.raises(TypeError):
         mount = Mount()
         assert isinstance(mount, Mount)
 
 
+@pytest.mark.with_mount
 class TestMount(object):
 
     """ Test the mount """
 
     @pytest.fixture(autouse=True)
-    def setup(self):
+    def setup(self, config):
 
-        self.config = load_config(ignore_local=True)
+        self.config = config
```

Comment:
Then the parking will be messed up. This test should be run when you have an actual mount attached and are wanting to watch what it does so it would in essence be testing your particular configuration.

Reply:
Ah, good point. So, if using real hardware (not just on PAN001), you want there to be a pocs_local.yaml file with an accurate location of the mount (OR read it from the mount... if the mount can provide that).

But, if not using real hardware, then we can test with several different locations.